src/components/Header.jsx

```diff
@@ -0,0 +1,9 @@
+export const Header = ({ text }) => {
+  return (
+ 		<header className="header-container">
+ 			<h3 className="header-text">
+                { text }
+            </h3>
+ 		</header>
+ 	);
+ };
```

Comment:
Can we also use `propTypes` here? It will be similar to how we did it in `Album.jsx`. This is so we can validate the props that we're passing into this component and will help us avoid bugs and errors when using this component later on.